cafebabel/templates/home.html

```diff
@@ -22,4 +22,16 @@
 {% endblock %}
 
 
-{% block body %}{% endblock %}
+{% block body %}
+  {% for article in articles %}
+    {% if loop.first %}
+      {# TODO(Johan): particular treatment? #}
```

Comment:
@cafebabel-webmaster for more options related to loops and indexes: http://jinja.pocoo.org/docs/2.10/templates/#for